front/app/containers/Admin/settings/general/Form.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { screen, render, userEvent } from 'utils/testUtils/rtl';
+import Form, { Props } from './Form';
+
+const orgNameEN = 'EN org name';
+const orgNameNL = 'NL org name';
+const defaultProps: Props = {
+  onSubmit: jest.fn(),
+  defaultValues: {
+    organization_name: { en: orgNameEN, 'nl-NL': orgNameNL },
+    locales: ['en' as const, 'nl-NL' as const],
+    organization_site: 'https://mywebsite.com',
+  },
+};
+
+// Needed for language selector of org name multiloc input
+jest.mock('hooks/useAppConfigurationLocales', () =>
+  jest.fn(() => ['en', 'nl-NL'])
+);
+
+describe('Form', () => {
```

Comment:
Love the test! 🤩 Maybe you can put something more concrete here like, Admin settings form, just to make it clearer.